src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/controller/RecommendationControllerTest.kt

```diff
@@ -192,6 +193,222 @@ class RecommendationControllerTest() : IntegrationTestBase() {
     assertThat(response.get("indexOffenceDetails")).isEqualTo(null)
   }
 
+  @Test
+  fun `update with manager recall decision`() {
+    mappaDetailsResponse(crn)
+    userAccessAllowed(crn)
+    allOffenderDetailsResponseOneTimeOnly(crn)
+    convictionResponse(crn, "011")
+    licenceConditionsResponse(crn, 2500614567)
+    releaseSummaryResponse(crn)
+    oasysAssessmentsResponse(crn)
+    deleteAndCreateRecommendation()
+    allOffenderDetailsResponseOneTimeOnly(crn = crn, delaySeconds = 0L, firstName = "Arthur")
+    updateRecommendation(
+      updateRecommendationRequest(),
+      "previousReleases, previousRecalls, mappa, indexOffenceDetails, convictionDetail, personOnProbation"
+    )
+    updateRecommendation(secondUpdateRecommendationRequest())
+
+    webTestClient.patch()
+      .uri("/recommendations/$createdRecommendationId/manager-recall-decision")
+      .contentType(MediaType.APPLICATION_JSON)
+      .body(
+        BodyInserters.fromValue(managerRecallDecisionRequest(decision = "RECALL"))
+      )
+      .headers { it.authToken(roles = listOf("ROLE_MAKE_RECALL_DECISION_SPO")) }
+      .exchange()
+      .expectStatus().isOk
+
+    webTestClient.get()
+      .uri("/recommendations/$createdRecommendationId")
+      .headers { it.authToken(roles = listOf("ROLE_MAKE_RECALL_DECISION")) }
+      .exchange()
```

Comment:
Do we need all these 100s of assertions here. In reality, this will be right at the start of process so shouldn't really be many fields set on a case (in the interest of trying to keep the line count of this test file at a manageable level!)

Reply:
Ok, refactored as suggested!